conftest.py

```diff
@@ -25,8 +25,21 @@ def set_fuser(fuser):
         torch._C._jit_override_can_fuse_on_gpu(True)
         torch._C._jit_set_texpr_fuser_enabled(True)
 
+def check_environment():
+    checks = [
+        # VAR_NAME, forbidden value
```

Comment:
shouldn't this read VAR_NAME, required value?

sounds backward to me.  IIUC you want to make sure logging is off, for instance?